.travis.yml

```diff
@@ -9,40 +9,36 @@ env:
   matrix:
     - ACTION=bandit
       OS=fedora
-      OS_VERSION=29
-      PYTHON_VERSION=2
-      ENGINE=docker
-    - ACTION=bandit
```

Comment:
@athos-ribeiro I'm not sure about this, is it worth to keep bandit for py2 as well? I vote for dropping bandit on py2

Reply:
Giving this a second thought,  we should drop F29 completely. Would it be possible to run bandit for py2 somewhere else? e.g., CentOS7? We could try to push the package to EPEL if it is not there :)

Reply:
Agreed, the package is installed using pip anyway so it should work for CentOS too